tests/test_bucketlist_resource.py

```diff
@@ -76,7 +81,61 @@ def test_cant_view_bucketlists_without_token(self):
         self.assertTrue(response.status_code == 401)
         self.assertTrue(b'Authorization Required' in response.data)
 
-    #view specific bucketlist
+    def test_user_cant_view_second_bucketlist_when_limit_is_one(self):
+        self.headers["Content-Type"] = "None"
```

Comment:
In the setUp for the tests I have set the content-type of requests to be application/json. Get requests do not require this setting and raise an error when the content type is set to application/json. So to clear the content-type defined in setup I have to reset it to None